SampleApp/SampleApp/Screens/DeviceRegistrationView.swift

```diff
@@ -15,22 +15,22 @@ final class DeviceRegistrationViewModel: ObservableObject {
     @Published var clientSecret = ""
     @Published var result = ""
 
-    private func configureRealifeTechSDK() {
+    func configureAndRegisterDevice() {
         let configuration = SDKConfiguration(
             appCode: appCode,
             clientSecret: clientSecret,
             apiUrl: "https://api-staging.livestyled.com/v3",
-            graphQLApiUrl: "https://staging-graphql-eu.realifetech.com")
+            graphQLApiUrl: "https://staging-graphql-eu.realifetech.com",
+            webOrderingJourneyUrl: nil)
         RealifeTech.configureSDK(with: configuration)
-    }
-
-    func registerDevice() {
-        configureRealifeTechSDK()
+        RealifeTech.setNotificationServiceExtensionWith(
```

Comment:
Is this optional? What happens if they choose not to use a service extension and so don't need to set it?

Reply:
Yes, this is optional. Only if integrator wants to use Notification Service Extension to track notifications received, they should call this method when app launch.
If they don't want to track notifications received, they don't need to call this method. And the other parts of initialise SDK are the same (configureSDK and register device).